src/renderer/components/+workloads-pods/pod-details-container.tsx

```diff
@@ -63,17 +64,16 @@ export class PodDetailsContainer extends React.Component<Props> {
         }
         {ports && ports.length > 0 &&
         <DrawerItem name={<Trans>Ports</Trans>}>
-          {
-            ports.map(port => {
-              const { name, containerPort, protocol } = port;
-              const key = `${container.name}-port-${containerPort}-${protocol}`
-              return (
-                <div key={key}>
-                  {name ? name + ': ' : ''}{containerPort}/{protocol}
-                </div>
-              )
-            })
-          }
+          <div>
+            {
+              ports.map((port) => {
+                return(
```

Comment:
same here

Reply:
Not sure what you mean here...

Reply:
I mean shortening to remove the return.

Reply:
ah, sorry, code has changed since then, not so obvious to do that now.